includes/class-wc-payments.php

```diff
@@ -82,6 +82,8 @@ public static function init() {
 			return;
 		}
 
+		add_action( 'admin_init', [ __CLASS__, 'add_woo_admin_notes' ] );
```

Comment:
That's a nice trick =)

One side-efect is that we'll be accessing the notes storage each time the hook fired (+1 select query, AFAIR).
We should be fine, as mysql will cache the query, so :shipit: